bus.css

```diff
@@ -0,0 +1,68 @@
+/*reset*/
+
+* {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html, body {
+  height: 100%;
+}
+body {
+  background-color: #F2EBC7;
+}
+.app {
+  width: 960px;
+  margin: 0 auto;
+  height: auto;
+  background-color: #962D3E;
+}
+h1 {
+  text-align: center;
+  font-family: 'Lobster Two', cursive;
+  font-size: 5em;
+}
+p {
+  font-family: Georgia, serif;
+  font-size: 2em;
+  text-align: center;
+  margin-top: 40px;
+  font-family: 'Lobster Two', cursive;
+}
+#restart {
+  display: none;
+  width: 150px;
+  height: 50px;
+  text-align: center;
+  font-size: 2em;
+  margin-left: 400px;
+  margin-top: 40px;
+  margin-bottom: 40px;
+  font-family: 'Lobster Two', cursive;
+  border-style: outset;
+  box-shadow: 3px 3px 3px 3px black;
+}
+img {
+  width: auto;
+  height: 200px;
+  margin-top: 75px;
+  margin-left: 80px;
```

Comment:
Using text-align: center on the parent container paired with margins on the img element might result in a smoother transition for the hover change.